docs/index.html

```diff
@@ -19,18 +19,20 @@
             </div>
             <div class="playback-controls">
                 <button id="pause-play">
-                    <span class="play" >&#x25BA;</span>
-                    <span class="pause" style="transform: translateY(-0.075em);">&#x23F8;</span>
+                    <span class="play" title="play">▶️</span>
```

Comment:
Leave these as html entities so they're easier to edit. Some of the emojis I have been using have to be forced into text mode, meaning they are made of 2 codepoints rather than 1.

Either that, or pop in some svg icons since those are more portable.